### DISCUSSION TITLE
Vote for lokax as RisingLight member

### DISCUSSION THREAD

**Discussion by xxchan** [risinglightdb/risinglight#675] [Member]
@lokax has made great contributions to the RisingLight. 

In #648, @lokax added support for pushing the filter pass the agg.

In #651, @lokax implemented `Having` and a more flexible `OrderBy`.

In #653, @lokax implemented column pruning for all operators, which is quite a lot of work.

In #659, @lokax implemented another cool feature: eliminate common sub-expressions.

@lokax demonstrated a good understanding of the RisingLight project. Therefore, I'd like to promote them as RisingLight member.

If you are a member of the RisingLight organization, you may vote for this proposal with 👍 and 👎. According to [GOVERNANCE](https://github.com/risinglightdb/risinglight/blob/main/GOVERNANCE.md), a minimum requirement of 2 votes are required for accepting this proposal. Thanks!

**Comment by skyzh** [comment]
With 3 votes from @skyzh, @xxchan and @likg227, we are accepting this vote. Welcome to the RisingLight community! @lokax
